tests/integration/utils.ts

```diff
@@ -0,0 +1,15 @@
+import { balena, BalenaSdk } from './setup';
+
+export const getInitialOrganization = async () => {
```

Comment:
NIT: would calling this `default` be more appropriate?

Reply:
Didn't do so, since I expect more helpers to be added here & also didn't have any issue while consuming this in the test.